### ISSUE TITLE
Реализовано ли в этого пакета ответы Successful с платежки?

### ISSUE THREAD

**Issue by fomvasss** [ange007/omnipay-interkassa#1]
Вижу что у вас в методе [isSuccessful](https://github.com/ange007/omnipay-interkassa/blob/master/src/Message/CompletePurchaseResponse.php)  всегда возвращает false, так должно быть 
 или это просто не реализовано?

**Comment by ange007** [comment]
> Вижу что у вас в методе isSuccessful всегда возвращает false, так должно быть
или это просто не реализовано?

Просто основной функционал прописан для этого в AbstractResponse, почему так - не помню :) но данная библиотека это по сути просто не совместимый форк https://github.com/hiqdev/omnipay-interkassa/ так как у них не было на тот момент никакой совместимости со стандартной логикой omnipay.

Вскоре возможно перепишу как нужно, так как будет потребность в новом проекте.

**Comment by fomvasss** [comment]
Да,  видил ваш Issues в https://github.com/hiqdev/omnipay-interkassa/, поэтому и взял ваш пакет. То ваш драйвер правильно обрабатывает ответ от интеркассы об успешном платиже? 

И было бы хорошо чтобы выш драйвер уже использовал omnipay v3, так как сейчас я использу Laravel + врапер для omnipay 3, и получается нужно делать форк на вашу либу

**Comment by ange007** [comment]
Да, у меня всё работает.
Но думаю тесты в любом случае не помешают.

> И было бы хорошо чтобы выш драйвер уже использовал omnipay v3, так как сейчас я использу Laravel + врапер для omnipay 3, и получается нужно делать форк на вашу либу

Ну вот как-раз будет повод судя по всему.
